lib/senders/email.js

```diff
@@ -226,12 +228,21 @@ module.exports = function (log) {
   Mailer.prototype.send = function (message) {
     log.trace({ op: 'mailer.' + message.template, email: message.email, uid: message.uid })
 
-    var localized = this.localize(message)
+    const localized = this.localize(message)
+
+    const template = message.template
+    let templateVersion = TEMPLATE_VERSIONS[template]
+    if (! templateVersion) {
+      log.error({ op: 'emailTemplateVersion.missing', template })
+      templateVersion = 1
+    }
+    message.templateVersion = TEMPLATE_VERSIONS[template]
```

Comment:
Should this use the value from `templateVersion` and not `TEMPLATE_VERSIONS[template]`